jwql/website/apps/jwql/monitor_pages/monitor_readnoise_bokeh.py

```diff
@@ -12,42 +12,55 @@
 
     ::
 
-        from jwql.website.apps.jwql import monitor_pages
+        .
         monitor_template = monitor_pages.ReadnoiseMonitor()
         monitor_template.input_parameters = ('NIRCam', 'NRCA1_FULL')
 """
 
 from datetime import datetime, timedelta
 import os
 
+from bokeh.embed import components
+from bokeh.layouts import column, row
+from bokeh.models import Panel, Tabs  # bokeh <= 3.0
+from bokeh.models import ColumnDataSource, HoverTool
+# from bokeh.models import TabPanel, Tabs  # bokeh >= 3.0
```

Comment:
Nice! I like notes for the future.

Reply:
Figured once this is in you or I can start taking a look at bumping the bokeh version!